ajax/libs/reactive-elements/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "reactive-elements",
+  "version": "0.6.7",
+  "description": "Allows to use React.js component as custom element",
+  "filename": "reactive-elements.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/PixelsCommander/ReactiveElements.git"
+  },
+  "keywords": [
```

Comment:
@cdnjs/intern3 can you guys "discuss" first? It's a shame that somebody says something that another one just do it without any discuss here as you all are still in a early learning phase.